nifi-nar-bundles/nifi-standard-services/nifi-hwx-schema-registry-bundle/nifi-hwx-schema-registry-nar/pom.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- 
+    Copyright 2016 Hortoworks, Inc. All rights reserved. 
```

Comment:
Sorry - you're right. I fixed this in all my java code but forgot that this stuff automatically gets added to XML files and the like as well.